lib/main.dart

```diff
@@ -16,8 +21,11 @@ class App extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      builder: DevicePreview.appBuilder,
-      home: Root()
-    );
+      title: welcomeMessage,
+      debugShowCheckedModeBanner: false,
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+      ),
```

Comment:
Do we need this?

Reply:
No.

Reply:
We could change it later according to needs